tests/test_ilastik/helpers/shellGuiTestCaseBase.py

```diff
@@ -150,12 +174,26 @@ def getPixelColor(self, imgView, coordinates, debugFileName=None, relativeToCent
 
         return img.pixel(point)
 
-    def moveMouseFromCenter(self, imgView, coords, modifier=Qt.NoModifier):
-        centerPoint = old_div(imgView.rect().bottomRight(), 2)
-        point = QPoint(*coords) + centerPoint
+    def moveMouseFromCenter(
+        self,
+        imgView: QAbstractScrollArea,
+        point: Union[QPointF, QPoint, Iterable[numbers.Real]],
+        modifier: int = Qt.NoModifier,
+    ):
+        """Move the mouse to a specific point.
+
+        Args:
+            imgView: View that will receive mouse events.
+            point: Target coordinate in relation to imageView center.
+            modifier: This modifier will be active when pressing, moving and releasing.
+
+        """
+        centerPoint = imgView.rect().bottomRight() / 2
+        point = _asQPointF(point) + centerPoint
```

Comment:
```suggestion
        absolutePoint = _asQPointF(point) + imgView.rect().center()
```

Reply:
hm, I oriented myself at the `strokeMouseFromCernter` method, that also uses `...Point` for the start and end points, despite them being kind of deltas, too...